.github/workflows/repository-and-gateway.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Start containers
         run: |
           docker pull arangodb:latest
-          npm run docker:test
+          npm run docker:test-up
```

Comment:
is docker compose down the right way to bring down the containers or should it be npm run docker:test-down ?

Reply:
I think it should be with npm since our command also deletes volumes. Didn't notice